File: src/logicServer/src/main/java/via/sep3/logicserver/shared/MemberDTO.java
package via.sep3.logicserver.shared;

import org.springframework.stereotype.Component;

import lombok.Data;

@Data
@Component
// In new build
public class MemberDTO {
    private int id;
    private String username;
    private byte[] password;
    private byte[] salt;
    private String position;
}

File: src/dataAccessServer/Shared/DTOs/MemberDTO.cs
namespace Shared.DTOs;

// In new build
public class MemberDTO
{
    public int Id { get; init; }
    public string? Username { get; set; }
    public byte[]? Password { get; set; }
    public byte[]? Salt { get; set; }
    public string? Position { get; set; }
}